docs/kb/compaction.rst

```diff
@@ -188,7 +188,31 @@ Which Strategy is best to use
 -----------------------------
 Use the table in :ref:`Which strategy is best <which-strategy-is-best>` to determine the right strategy for your needs.
 
+.. _compaction-off-strategy-compaction:
 
+Off-strategy Compaction
+----------------------------
+
+Off-strategy compaction is a default mechanism that improves the efficiency of node operations, 
+such as ``bootstrap``, ``repair``, or ``replace``. It keeps SStables created by node operations 
+in a separate “maintenance” data set ignored by the regular compaction (STCS, LCS, etc.).
+
+After the node operation is completed, all the new SSTables kept in the maintenance set are 
+reshaped and then integrated into the main set of SStables to fall under the regular compaction strategy. 
+
+Off-strategy compaction prevents creating excessive compaction backlog due to a large number 
+of SStables generated by node operations. Compaction work during node operations is limited, 
+which speeds up the completion of ``repair``, ``bootstrap``, and others. 
+
+As a rule, we recommend *against* disabling off-strategy compaction. However, if needed, it can be 
```

Comment:
AFAIU, "nodetool stop reshape" does not disable off-strategy compaction, its abort teh current running compaction

Reply:
@raphaelsc is it the case?

Reply:
nodetool stop reshape aborts the off-strategy on behalf of a table, but without throwing out the partial results. off-strategy can still continue on behalf of other tables needing off-strategy.